<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>287</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>添加药品</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/back.jpg);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>新增药品名称</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="nameLine">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>药品进价</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="InPrice">
     <property name="inputMethodHints">
      <set>Qt::ImhDigitsOnly|Qt::ImhPreferNumbers</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>药品处方价</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="OutPrice">
     <property name="inputMethodHints">
      <set>Qt::ImhDigitsOnly|Qt::ImhPreferNumbers</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>进货量</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="InStock">
     <property name="inputMethodHints">
      <set>Qt::ImhDigitsOnly|Qt::ImhPreferNumbers</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>最小库存阈值设置</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="minStock">
     <property name="inputMethodHints">
      <set>Qt::ImhDigitsOnly|Qt::ImhPreferNumbers</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>最大库存阈值设置</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="maxStock">
     <property name="inputMethodHints">
      <set>Qt::ImhDigitsOnly|Qt::ImhPreferNumbers</set>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="background_image.qrc"/>
 </resources>
 <connections/>
</ui>
